src/libraries/System.Diagnostics.DiagnosticSource/src/ILLink/ILLink.Substitutions.Shared.xml

```diff
@@ -0,0 +1,10 @@
+<linker>
+  <assembly fullname="System.Diagnostics.DiagnosticSource">
+    <type fullname="System.Diagnostics.Metrics.Meter" feature="System.Diagnostics.Metrics.Meter.IsSupported" featurevalue="false">
+      <method signature="System.Boolean get_IsSupported()" body="stub" value="false" />
+    </type>
+    <type fullname="System.Diagnostics.Metrics.Instrument" feature="System.Diagnostics.Metrics.Meter.IsSupported" featurevalue="false">
+      <method signature="System.Boolean get_Enabled()" body="stub" value="false" />
+    </type>
+  </assembly>
```

Comment:
(nit) you can refactor this to avoid the duplication.

```suggestion
  <assembly fullname="System.Diagnostics.DiagnosticSource" feature="System.Diagnostics.Metrics.Meter.IsSupported" featurevalue="false">
    <type fullname="System.Diagnostics.Metrics.Meter">
      <method signature="System.Boolean get_IsSupported()" body="stub" value="false" />
    </type>
    <type fullname="System.Diagnostics.Metrics.Instrument">
      <method signature="System.Boolean get_Enabled()" body="stub" value="false" />
    </type>
  </assembly>
```